src/metrics/metric_types.go

```diff
@@ -20,3 +24,61 @@ func (qd QueryDefinition) GetDataModels() interface{} {
 	ptr := reflect.New(reflect.ValueOf(qd.dataModels).Type())
 	return ptr.Interface()
 }
+
+func (qd *QueryDefinition) insertDatabaseNames(databases collection.DatabaseList) *QueryDefinition {
+	databaseList := ""
+	for database := range databases {
+		databaseList += `'` + database + `',`
+	}
+	databaseList = databaseList[0 : len(databaseList)-1]
+
+	qd.query = strings.Replace(qd.query, `%DATABASES%`, databaseList, 1)
+
+	return qd
+}
+
+func (qd *QueryDefinition) insertSchemaTables(schemaList collection.SchemaList) *QueryDefinition {
+	schemaTables := make([]string, 0)
```

Comment:
perhaps also unify this way to create the string of concatenated things with the others `insert` methods?

Reply:
Done also 👍